src/RA_Dlg_MemBookmark.cpp

```diff
@@ -540,30 +537,32 @@ void Dlg_MemBookmark::SetupColumns(HWND hList)
     //	Remove all data.
     ListView_DeleteAllItems(hList);
 
-    LV_COLUMN col;
-    ZeroMemory(&col, sizeof(col));
-
-    for (size_t i = 0; i < NumColumns; ++i)
+    auto idx{ 0 };
+    for (auto& sColTitle : ra::COLUMN_TITLE)
     {
-        col.mask = LVCF_TEXT | LVCF_WIDTH | LVCF_SUBITEM | LVCF_FMT;
-        col.cx = COLUMN_WIDTH[i];
-        ra::tstring colTitle = NativeStr(COLUMN_TITLE[i]).c_str();
-        col.pszText = const_cast<LPTSTR>(colTitle.c_str());
-        col.cchTextMax = 255;
-        col.iSubItem = i;
-
-        col.fmt = LVCFMT_CENTER | LVCFMT_FIXED_WIDTH;
-        if (i == NumColumns - 1)
+        ra::tstring tszText{ sColTitle };
+        LV_COLUMN col
+        {
+            col.mask       = LVCF_TEXT | LVCF_WIDTH | LVCF_SUBITEM | LVCF_FMT,
+            col.fmt        = LVCFMT_CENTER | LVCFMT_FIXED_WIDTH,
+            col.cx         = ra::COLUMN_WIDTH.at(idx),
+            col.pszText    = tszText.data(),
+            col.cchTextMax = 255,
+            col.iSubItem   = idx
+        };
+
+        if (idx == (ra::COLUMN_TITLE.size() - 1))
             col.fmt |= LVCFMT_FILL;
 
-        ListView_InsertColumn(hList, i, (LPARAM)&col);
+        ListView_InsertColumn(hList, idx, &col);
+        idx++;
     }
-
     m_nNumOccupiedRows = 0;
 
-    BOOL bSuccess = ListView_SetExtendedListViewStyle(hList, LVS_EX_FULLROWSELECT | LVS_EX_GRIDLINES | LVS_EX_DOUBLEBUFFER);
-    bSuccess = ListView_EnableGroupView(hList, FALSE);
```

Comment:
is EnableGroupView not required?

Reply:
I didn't see any different effect. It was false and it returned 0 meaning it was already disabled. Enabling it by putting true instead, I couldn't add/remove bookmarks.